### ISSUE TITLE
new upgrade: regenerative passive overshield

### ISSUE THREAD

**Issue by 0xDE57** [0xDE57/SpaceProject#58]
Passive Regen Shield [1,2,3,4] - rechargeable overshield -> some forgiveness

Levels: [1,2,3,4] 
Protection:    25, 50, 75, 100
Cooldown: 20s,??,10s?

Take the edge off the initial hit with a rechargable overhshield! Passive means you don't even have to do anything!
It will take damage before the hull. Any extra damage beyond the shield value is passed onto the hull.
When overshield is fully depleted, it will break/overheat and require a long cooldown before reactivating.

Like the active shield, this will protect the ship.

**Comment by webbrain-one** [comment]
Hi! I've opened a pull request to implement the regenerative passive overshield described here. You can check it out here: https://github.com/0xDE57/SpaceProject/pull/65

**Comment by 0xDE57** [comment]
@webbrain-one 

No, you have not. You only added boilerplate copy pasta AI slop based on the existing structure. You have not implemented anything.

You are mass opening automated, low-effort PR's against many repo's. Likely an advertising campaign rather than authentic activity.

Fuck off.
